src/main/java/Duke.java

```diff
@@ -1,10 +1,133 @@
 public class Duke {
+
+    /**
+     * The main method is the entry point of the program. It prompts the user for their name,
+     * instantiates a TaskMaster object, and handles user commands.
+     *
+     * @param args Command-line arguments passed to the program.
+     */
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+
+        showWelcomeMessage();
+        showHelpMessage();
+
+        IO inOut = new IO();
+
+        String userName = getUserName(inOut);
+
+        TaskMaster compileItems = new TaskMaster();
```

Comment:
Good job on using plural name compileItems to represent a collection of objects!